lib/gringotts/gateways/global_collect.ex

```diff
@@ -14,41 +14,42 @@ defmodule Gringotts.Gateways.GlobalCollect do
   | Refund                       | `refund/3`    |
   | Void                         | `void/2`      |
 
-  ## Optional or extra parameters
+  ## Optional parameters
 
   Most `Gringotts` API calls accept an optional `Keyword` list `opts` to supply
   optional arguments for transactions with the gateway.
 
-  | Key                      | Status          |
+  | Key                      | Remark          |
   | ----                     | ---             |
-  | `merchantCustomerId`     |  implemented    |
-  | `description`            |  implemented    |
-  | `customer_name`          |  implemented    |
-  | `dob`                    |  implemented    |
-  | `company`                |  implemented    |
-  | `email`                  |  implemented    |
-  | `phone`                  |  implemented    |
-  | `order_id`               |  implemented    |
-  | `invoice`                |  implemented    |
-  | `billingAddress`         |  implemented    |
-  | `shippingAddress`        |  implemented    |
-  | `name`                   |  implemented    |
-  | `skipAuthentication`     |  implemented    |
+  | `merchantCustomerId`     |  Identifier for the consumer that can be used as a search criteria in the Global Collect Payment Console |
+  | `description`            |  Descriptive text that is used towards to consumer, either during an online checkout at a third party and/or on the statement of the consumer |
+  | `dob`                    |  The date of birth of the consumer Format: YYYYMMDD   |
+  | `company`                |  Name of company, as a consumer                       |
+  | `email`                  |  Email address of the consumer                        |
+  | `phone`                  |  Phone number of the consumer                         |
+  | `invoice`                |  Object containing additional invoice data            |
```

Comment:
The schema of this `Object` is not documented.

Reply:
Structure of invoice object is shown in the examples if you read further in this file along with billingAddress,  shippingAddress and name.In addition to this, I am adding a reference to the link showing the required fields structure in Globalcollect docs.

Reply:
I did notice it's presence in the examples, but an example cannot define the schema. The `invoice` example is:

Reply:
Ya, I agree that examples can't define the schema so, for more information, I have added a link to refer.